core/blockdag/blockdag_test.go

```diff
@@ -308,15 +306,6 @@ func CalcBlockWeight(blocks int64, blockhash *hash.Hash, state byte) int64 {
 	return 1
```

Comment:
I think the mock logic in `CalcBlockWeight` might need to improve even in the unit test. the subsidy calculation is too important to ignore in the blockdag consensus test.